pages/doc/wavefront_spring_boot3.md

```diff
@@ -0,0 +1,450 @@
+      ```
+1. Add data to your application before you start to view the data on Wavefront.
+    {% include tip.html content="Try out the [Wavefront for Spring Boot 3 Tutorial](wavefront_springboot3_tutorial.html)."%}
+1. Click the link (for example, `https://wavefront.surf/us/<name>`) and you are taken to the Wavefront Spring Boot Inventory dashboard where you can examine the data sent by your application.
+    <br/>Example:
+    ![Spring Boot metrics dashbaord](images/springboot_metrics.png)
+    If your application uses trace data, click the link in the **Tracing** section of the dashboard to be directed to the Wavefront Traces Browser.
+    <br/>Example:
+    ![Spring Boot traces browser](images/springboot_span_logs_pet_clinic.png)
+    {% include note.html content="Want to see the cool information you can gather from the Traces Browser? See [Explore the Traces Browser](tracing_traces_browser.html)." %}
+<!---
+<iframe width="768" height="432" src="https://www.youtube.com/embed/K-cviV9mKKA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+--->
+
+
+### Custom Configurations
+
+Add the following custom configurations to the `application.properties` file.
+
+<p><span style="font-size: large; font-weight: 500">Invite Users</span></p>
+
+You can invite users and let them send data to the same cluster:
+
+1. Click the link that was printed on your console and navigate to the Wavefront Service Dashboard:
+    {% include note.html content="Save the link that you used to access the Service Dashboard." %}
+    1. Click the gear icon and select **Accounts**.
+    1. Click **Invite New Users** and specify a comma-separated list of email addresses.<br/>
+        ![Invite Users](/images/spring_boot_invite_users.png)
+  The users get an email with a link to reset their password. They can then access your dashboard.
+1. Information about the token and URL are displayed on your terminal. Add them to your project’s `application.properties` file.
+    ```
+    management.wavefront.api-token=<Enter_Token>
+    management.wavefront.uri=https://wavefront.surf
+    ```
+1. If you are using the freemium account and want the single-use login URL to show on the terminal each time you start the application, add `wavefront.freemium-account` to the `application.properties` file and set it to `true`.
+  ```
+  wavefront.freemium-account=true
+  ```
+    {% include tip.html content="If you don’t want Wavefront to auto-negotiation a freemium account for you, set the value to `false`."%}
+1. Restart your application.
+
+<a name="proxy"></a>
+<p><span style="font-size: large; font-weight: 500">Use the Wavefront Proxy</span></p>
+
+The [Wavefront proxy](proxies.html) ingests data and forwards them to the Wavefront service in a secure, fast, and reliable manner. It prevents data loss, simplifies firewall configuration, and allows you to filter or enrich data before they arrive at the Wavefront service.
```

Comment:
i'd suggest that we use "data" in singular, not plural

... ingests data and forwards is...

...or enrich data before it arrives at the Wavefront service. ...